network/core-node/admin-guide/installation.mdx

```diff
@@ -3,56 +3,62 @@ title: Installing
 
-SDF maintains a [quickstart image](https://github.com/stellar/docker-stellar-core-horizon) that bundles Stellar Core with Horizon and postgreSQL databases. It's a quick way to set up a default, non-validating, ephemeral configuration that should work for most developers.
+## Release Version
 
-In addition to SDF images, Satoshipay maintains separate Docker images for [Stellar Core](https://github.com/satoshipay/docker-stellar-core) and [Horizon](https://github.com/satoshipay/docker-stellar-horizon). The Satoshipay Stellar Core Docker image comes in a few flavors, including one with the AWS CLI installed and one with the Google Cloud SDK installed. The Horizon image supports all Horizon environment variables.
+Whichever method you use, you should make sure to install the latest [release](https://github.com/stellar/stellar-core/releases) since these builds are backwards compatible and are cumulative.
 
-### Production environments
+The version number scheme that we follow is `protocol_version.release_number.patch_number`, where:
 
-The SDF also maintains a Stellar-Core-only standalone image: [`stellar/stellar-core`](https://hub.docker.com/r/stellar/stellar-core).
+- `protocol_version` is the maximum protocol version supported by that release (all versions are 100% backward compatible),
+- `release_number` is bumped when a set of new features or bug fixes not impacting the protocol are included in the release, and
+- `patch_number` is used when a critical fix has to be deployed.
+
+## Installing From Source
+
+The Stellar Core source code repository contains extensive and thorough instructions to build the software from source. Please [check out `INSTALL.md`](https://github.com/stellar/stellar-core/blob/master/INSTALL.md) for more information.
+
+## Package-Based Installation
+
+If you are using a recent LTS version of Ubuntu, we provide the latest stable releases of [`stellar-core`](https://github.com/stellar/stellar-core) and [`stellar-horizon`](https://github.com/stellar/go/tree/master/services/horizon) in Debian binary package format.
+
+You may choose to install these packages individually, which offers the greatest flexibility, but requires **manual** creation of the relevant configuration files and configuration of a **PostgreSQL** database.
+
+## Docker-Based Installation
+
+### Development Environments
+
+SDF maintains a [quickstart image](https://github.com/stellar/quickstart) that bundles Stellar Core with Horizon and the necessary PostgreSQL databases. It's a quick way to set up a default, non-validating, ephemeral configuration that should work for most developers. Additionally, the quickstart image can be spun up pre-configured for use as a Mainnet, Testnet, Futurenet, or Local network Node.
```

Comment:
I think this is up-to-date as far as a description of quickstart goes. I think this could use some cleanup of the language, and make it more clear that it's not intended as a production-level "appliance."

Reply:
Cool deal.  Thanks for checking!